values.yaml

```diff
@@ -0,0 +1,26 @@
+# Default resource requests and limits used by the Tonic services.
+resources:
```

Comment:
It's probably better if we move each of these into their existing definitions under tonicai, e.g.:

```yaml
tonicai:
  notifications:
    resources: ...
```

I've been regretting my decision to hoist the GPU resource to a top level key since I merged it and this seems like a good opportunity to address it.

Reply:
Additionally, when testing these changes with an existing values file I have, I would receive errors when not using this file which is an existing use case we need to consider. To address it, I believe including at the top of each deployment, adjusting service and defaults accordingly:

```
{{- $resources := coalesce (((.Values).resources).pyml_service) (dict "limits" (dict) "requests" (dict "memory" "512Mi" "ephemeral-storage" "1Gi")) -}}
```

And then using `{{ toYaml $resources | ... }}` instead of templating the `.Values....` access directly.

Reply:
I agree: having the resources at the top-level definitely feels... weird. I haven't yet tested your suggestion, but it looks like a reasonable step that removes the need for defaults in the `values.yaml`.